### ISSUE TITLE
Batch creation with API create 

### ISSUE THREAD

**Issue by jensens** [edutap-eu/edutap.wallet_google#4]
The Google create (POST) endpoint supports creation of multiple classes or passes at once. 
Nice to have: Support this 
see https://developers.google.com/wallet/generic/resources/performance-tips#batch-requests

**Comment by loechel** [comment]
Looking more into the batch processing that makes absolute sense.

we should have a separate batch.py module that creates either for all or for the relevent methods:
* create
* read
* update
* message
subrequests that than cann be added to a sync or async batch method.

this than pobably will have two internal helper methods build_batch and parse_batch_resopnse 

I do not see any problems to implement that, will give it a chance after #61 is merged.
